cypress/integration/metaSpec.js

```diff
@@ -6,18 +6,16 @@ describe('On a page with meta tag imgix paramaters', () => {
   context('The images on the page', () => {
     beforeEach(() => {
       cy.fixture('config.js').as('config');
-      // expect the number of image elements on the page to equal fixture's
-      cy.get('img').should('have.length', 4);
```

Comment:
This felt oddly specific and broke anytime I added an image to test with to the fixture. I decided to remove it rather than change this number every so often.

Reply:
Yeah I agree this is better left out. The only reason you usually have things like this is to prevent false positives in the tests because there are no images to match, so then all the `cy.each` calls run on zero images, so no assertions happen. If you wanted to, you could add an assertion that there is _at least_ 1 image, which should prevent against this case, but it's not really a big deal